<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExcludeParts</class>
 <widget class="QWidget" name="ExcludeParts">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>347</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Exclude parts</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QListView" name="lv_part_list">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="spacing">
      <number>5</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
